src/sentry/incidents/grouptype.py

```diff
@@ -353,4 +359,5 @@ class MetricIssue(GroupType):
                 },
             },
         },
+        hooks=DetectorLifeCycleHooks(pending_delete=delete_data_in_seer_for_detector),
```

Comment:
You're probably aware but just in case - these got removed in https://github.com/getsentry/sentry/pull/102990 in favor of deleting in the validators https://github.com/getsentry/sentry/pull/102919 sorry you'll have to rebase :(